src/app/components/Image/index.test.jsx

```diff
@@ -0,0 +1,94 @@
+import React from 'react';
+import snapshotTestHelper from '../../../__test__/snapshotTestHelper';
+import Image from './index';
+
+describe('Image', () => {
+  describe('with no data', () => {
+    snapshotTestHelper.shouldMatchSnapshot(
+      'should not render anything',
+      <Image />,
+    );
+  });
+
+  describe('with data', () => {
+    const imageData = arrayOfBlocks => ({
+      blocks: [
+        {
+          blockId: '',
+          type: 'image',
+          model: {
+            blocks: arrayOfBlocks,
+          },
+        },
+      ],
+    });
+
+    const rawImageBlock = {
+      blockId: '',
+      type: 'rawImage',
+      model: {
+        width: 640,
+        height: 420,
+        locator:
+          '/cpsprodpb/439A/production/_100960371_syrians_and_asylum_v2-nc.png',
+        originCode: null,
+        copyrightHolder: 'BBC',
+      },
+    };
+
+    const textBlock = (type, text) => ({
+      blockId: '',
+      type,
+      model: {
+        blocks: [
+          {
+            blockId: '',
+            type: 'text',
+            model: {
+              blocks: [
+                {
+                  type: 'paragraph',
+                  blockId: '',
+                  model: {
+                    text,
+                  },
+                },
+              ],
+            },
+          },
+        ],
+      },
+    });
+
+    const data = imageData([rawImageBlock]);
+
+    snapshotTestHelper.shouldMatchSnapshot(
```

Comment:
Do we _want_ to render an image if there is no alt text?

Reply:
It is valid to render an image with an empty alt attribute. Our renderer should handle this case, since journalists won't always remember to add it in.

Reply:
@sar It's not going to be compulsory for journalists to add alt text like it currently is in CPS?

Reply:
@sareh  It's not going to be compulsory for journalists to add alt text like it currently is in CPS?

Reply:
@sareh I just checked with @katikaa and it's going to be a mandatory field to add alt text in Optimo. For some reason if we don't receive alt text, I would to not render the image...